google-cloud-spanner/src/test/java/com/google/cloud/spanner/connection/DdlBatchTest.java

```diff
@@ -157,25 +157,6 @@ public void testExecuteCreateDatabase() {
                     .parse(Statement.of("CREATE DATABASE foo"))));
   }
 
-  @Test
-  public void testExecuteMetadataQuery() {
```

Comment:
This test specifically checked whether a DDL batch could execute an internal metadata query. That is no longer needed, as a DDL batch no longer handles that special case itself. Instead, this is handled directly in the connection for all types of transactions.